diff --git a/CMakeLists.txt b/CMakeLists.txt
index 62ed676..5326bad 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,6 +15,10 @@ include(GNUInstallDirs)
 set(CMAKE_CXX_STANDARD 11)
 
 # Third Party Dependencies
+if(EXISTS $ENV{CONDA_PREFIX})
+  message("CONDA PREFIX: $ENV{CONDA_PREFIX}")
+  list(APPEND CMAKE_FIND_ROOT_PATH $ENV{CONDA_PREFIX})
+
 find_package(GSL           REQUIRED)
 find_package(Eigen3 3.3    REQUIRED NO_MODULE)
 find_package(Python        REQUIRED COMPONENTS Development)
