diff --git CMakeLists.txt CMakeLists.txt
index 0f810b2..34ceea6 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -25,7 +25,7 @@ endif()
 
 find_package(GSL           REQUIRED)
 find_package(Eigen3 3.3    REQUIRED NO_MODULE)
-find_package(Python        REQUIRED COMPONENTS Development)
+find_package(Python 3.6 EXACT REQUIRED COMPONENTS Development)
 find_package(nlohmann_json REQUIRED)
 
 # Library setup
